### ISSUE TITLE
Remove redundant ci job

### ISSUE THREAD

**Issue by aahmed-se** [apache/pulsar#7754]
As part of ci fixes I removed all tasks in ci-unit-flaky.yaml , a dummy template still exists because it was are required job in github.
That needs to changed so the workflow can be deleted.

".github/workflows/ci-unit-flaky.yaml"

**Comment by jiazhai** [comment]
@zymap have you already opened a ticket in ASF Infra for this issue?

**Comment by zymap** [comment]
Yes. JIRA issue: https://issues.apache.org/jira/browse/INFRA-20708
